rosetta-ide/src/main/java/com/regnosys/rosetta/ide/server/RosettaFormattingOptionsRetriever.java

```diff
@@ -0,0 +1,13 @@
+package com.regnosys.rosetta.ide.server;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.lsp4j.FormattingOptions;
+import org.eclipse.lsp4j.jsonrpc.services.JsonRequest;
+import org.eclipse.lsp4j.services.LanguageServer;
+
+public interface RosettaFormattingOptionsRetriever extends LanguageServer{
```

Comment:
Usually something like this is called a `Provider`. :) Although, since it also extends `LanguageServer`, and we might use this interface to drop any other extension we might wanna make, I think it would be fine to just call this `RosettaLanguageServer`, as a catch-all for all future extensions.